packages/docs/docs/exporting-from-figma.md

```diff
@@ -0,0 +1,71 @@
+---
+id: exporting-from-figma
+title: Exporting designs from Figma to Remotion
+---
+
+Say you're collaborating as a Frontend Engineer with a design team, and you want a way to get a real-time representation of some of the design assets you have on Figma.
+
+You can export a design from Figma as an SVG file and import it as a React component in Remotion.
+
+## Open the Figma design
+
+The design we'll consider here is a suite of 3D graphics made by [Alzea Arafat](https://dribbble.com/alzea) for the design community. You can find them [here](<https://www.figma.com/file/VCoVolpndRmxkSd5AvctYZ/SALY---3D-Illustration-Pack-(Community)?node-id=7%3A4>). When you click the link to the design, click on the **"Library"** &mdash; in the sidebar &mdash; text to access all 3D graphics
+
+![Thumbnail](/static/img/export-figma/banner.png)
+
+Click on any of the vector graphics. Make sure you're selecting the 3D element alone. Don't select the whole container, the image below illustrates this.
```

Comment:
Why emphasize to only select the 3D element? If I import mockups from Figma, I actually do the opposite and select the whole frame, because then I can match the dimensions with the video.